### PR TITLE
arXiv dataset added

### PR DESCRIPTION
**Pull request by tanmoyio**

### CODE REVIEW

**Comment by tanmoyio** [comment]
Need help

File: datasets/arxiv_dataset/arxiv_dataset.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                    path_to_manual_file, _FILENAME, self.manual_download_instructions
+                )
+            )
+        return [datasets.SplitGenerator(name=datasets.Split.TRAIN, gen_kwargs={"path": path_to_manual_file})]
+
+    def _generate_examples(self, path=None, title_set=None):
+        """ Yields examples. """
+
+        data = []
+        for line in open(path, "r"):
```
Comment (cceyda) [inline datasets/arxiv_dataset/arxiv_dataset.py:120]: it is complaining about not having  `encoding="utf-8"` defined change line to -> `for line in open(path,  encoding="utf-8"):`

Comment (cceyda) [reply datasets/arxiv_dataset/arxiv_dataset.py:120]: btw why are you loading json line by line. Look at the example here -> https://github.com/huggingface/datasets/blob/master/templates/new_dataset_script.py#L162

Comment (tanmoyio) [reply datasets/arxiv_dataset/arxiv_dataset.py:120]: @cceyda I have fixed the code and still it is showing this error
Screenshot (21)

**Pull request closed without merging**